src/CalcManager/CalculatorManager.cpp

```diff
@@ -117,6 +117,15 @@ namespace CalculationManager
         m_displayCallback->SetParenDisplayText(parenthesisCount);
     }
 
+    /// <summary>
+    /// Callback from the engine
+    /// Used to set the narrator text when no parenthesis can be added
```

Comment:
Let's not say what this is used for.  At this level of the code, we are simply bubbling an event and have no idea what the resulting behavior will be.

Reply:
It looks like I removed it from the method above, but the comment applies to both methods, so now it is removed from both.

---
In reply to: [265407878](https://github.com/Microsoft/calculator/pull/284#discussion_r265407878) [](ancestors = 265407878)